### PR TITLE
docs: document php-fpm path exclusion from access logs

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

**Comment by dunglas** [comment]
Thanks for the PR, but we try to keep these files as small as possible. Maybe just a comment like "expose a URL for health checks"?

Users should refer to FPM docs for more details.

**Comment by thePanz** [comment]
@dunglas changed to `Exclude the /ping path from access logs (directive available for PHPv8.2)`

WDYT?

ps: please squash the commits from this PR :)

File: docker/php/php-fpm.d/zz-docker.conf [hunk 1]
```diff
@@ -6,4 +6,6 @@ process_control_timeout = 20
 listen = /var/run/php/php-fpm.sock
 listen.mode = 0666
 ping.path = /ping
+
+; Exclude the /ping path from access logs (directive available for PHPv8.2)
```
Comment (maxhelias) [inline docker/php/php-fpm.d/zz-docker.conf:10]:
```suggestion
; Exclude the /ping path from access logs (directive available since PHP 8.2)
```

**Comment by dunglas** [comment]
I stil doubt this comment is worth it.

**Comment by thePanz** [comment]
It helps if a user is personalizing the docker image (example: using PHP v8.1) and would help to quickly identify the failure of starting PHP-FPM.

Why a user would need to personalize the docker image with an older version of PHP?
- users might use the docker image for local development, while keeping the specs similar to the staging/production deployments which do not use docker

**Comment by dunglas** [comment]
PHP 8.1 will be EOLed on 25 Nov 2023, this is a very temporary patch :)

**Comment by thePanz** [comment]
True, but do not forget about all of those *legacy* projects that can not upgrade ;-)

FYI: https://packagist.org/packages/monolog/monolog/php-stats?ref=blog.packagist.com php 7.x is still out there, running on 35% of the monolog installs (monolog is just an example) :)

**Comment by maxhelias** [comment]
With the rework of the template for FrankenPHP, this file no longer exists.

**Pull request closed without merging**